cmd/krel/cmd/changelog.go

```diff
@@ -92,15 +91,11 @@ const (
 )
 
 func init() {
-	const (
-		tagFlag   = "tag"
-		tarsFlag  = "tars"
-		tokenFlag = "token"
-	)
+	const tagFlag = "tag"
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.bucket, "bucket", "kubernetes-release", "Specify gs bucket to point to in generated notes")
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.tag, tagFlag, "", "The version tag of the release, for example v1.17.0-rc.1")
```

Comment:
@saschagrunert -- Can we do the same here?

Reply:
Yes, sure :+1: